pkg/kubelet/volumes.go

```diff
@@ -243,7 +259,7 @@ func (kl *Kubelet) getPodVolumesFromDisk() map[string]cleanerTuple {
 				continue
 			}
 			if unmounter == nil {
-				glog.Errorf("Could not create volume unmounter for %s: %v", volume.Name, errUnsupportedVolumeType)
+				glog.Errorf("Could not create volume unmounter for %s: %v", volume.Name, createVolumeTypeError(volume.Name))
```

Comment:
That makes sense.. especially after rereading the error message :). I am ok with this as is